tests/integration/cli/detection/test_tiling_instseg.py

```diff
@@ -34,11 +37,28 @@
         "4",
         "--tiling_parameters.enable_tiling",
         "1",
+        "--tiling_parameters.enable_tile_classifier",
```

Comment:
Shouldn't we check the non-tile-classifier cli test?

Reply:
I could run additional testing, but doing so would double the testing time. My reasoning is that since enabling the tile classifier already covers the tiling test, running extra tests is unnecessary.